_posts/en/newsletters/2020-11-18-newsletter.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'Bitcoin Optech Newsletter #124'
+permalink: /en/newsletters/2020/11/18/
+name: 2020-11-18-newsletter
+slug: 2020-11-18-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains a warning about backdoored VM images.
+Also included are our regular sections with updates-about various projects.
+
+## Action items
+
+- **Backdoored VM images:** a user on Reddit [posted][reddit vm post]
+  about losing funds after using an AWS image that came with a Bitcoin
+  full node already installed and synced to a recent block.  Although
+  the source of the loss was not fully determined in the thread, it was
+  suggested that virtual machine images or other curated collections of
+  software, especially those designed for Bitcoin or other
+  cryptocurrency uses, provide an effective mechanism for delivering
+  backdoored software to valuable servers.  This is a reminder that you
+  should only install software from trustworthy sources.  Additionally,
+  please remember that your VM provider and their support staff can
+  likely access any private keys on your server even if you perfect
+  every other aspect of your security.  In short, please consider
+  performing extra diligence on any software or service to which you
+  will entrust the creation of non-reversible Bitcoin transactions.
+
+## News
+
+*No significant Bitcoin technical news this week.*
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.9.2rc1][C-Lightning 0.9.2] is a release candidate for
+  the next maintenance version of C-Lightning.  It contains new
+  features, updated options, and bug fixes.
+
+FIXME:harding to update section on Tuesday
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #4168][] hooks FIXME:dongcarl
+
+- [C-Lightning #4171][] updates the `hsmtool` command with a new
+  `dumponchaindescriptors` parameter that prints the [output script
+  descriptors][topic descriptors] for the keys and scripts used by
+  C-Lightning's onchain wallet.  These descriptors may then be imported
+  into a watch-only wallet to monitor for any onchain transactions made
+  by the LN node.  For example, this feature was [requested][c-lightning
+  #4110] to possibly help improve integration between [BTCPay
```

Comment:
I stumbled a bit on this extraordinary hypotheticality. It implies to me that the reason is doubtful or odd which then is not elaborated, though. What do you think about shortening that to "[…] this feature was requested to ease integration between […]"?

Reply:
Agree. Remove 'possibly'.

Reply:
You could go even further and remove "help" as well.

Reply:
Making the other changes, but I think "help" is warranted because the change itself does not improve the integration---it just makes improving the integration easier.